Example/StanwoodAnalytics/AppDelegate.swift

```diff
@@ -14,19 +14,12 @@ import UserNotifications
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
-    var viewController: ViewController?
-    let appController = AppController()
+    var appController: AppController?
 
     func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
 
-        if let navigationController = window?.rootViewController as? UINavigationController {
-
-            let viewController = navigationController.viewControllers.first as! ViewController
-            // let notificationDelegate = viewController as UNUserNotificationCenterDelegate
-            self.viewController = viewController
-            viewController.appController = appController
-        }
-
+        appController = AppController(window: window!)
+        appController?.configure()
```

Comment:
why not leave the call to configure() inside init(window:) but before the guard statements?

Reply:
Because it cannot call 
actions = Actions(appController: self) 
before it returns from init.